ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -78,7 +78,7 @@ interface {{ port_channel_interface.name }}
 {%     if port_channel_interface.phone.trunk is arista.avd.defined %}
    switchport phone trunk {{ port_channel_interface.phone.trunk }}
 {%     endif %}
-{%     if port_channel_interface.mode is arista.avd.defined("trunk") %}
+{%     if port_channel_interface.mode is arista.avd.defined %}
```

Comment:
To make this less "breaking" I think we should avoid printing "access". We can add that later in 5.0

```suggestion
{%     if port_channel_interface.mode is arista.avd.defined and port_channel_interface.mode != "access" %}
```